fix_mirrors.sh

```diff
@@ -74,6 +85,8 @@ function main () {
 	tmpdir=$(mktemp -d /tmp/XXX)
 	pushd $tmpdir
 	fuel_allow_noncontroller_deployment
+        fuel_allow_nonmongo_deployment
+        restart_nailgun
```

Comment:
This will restart nailgun no everytime the script is runs. I don't like the idea.

try to use restart_nailgun function inside fuel_allow_nonmongo_deployment and fuel_allow_noncontroller_deployment when match is found.